client/components/domains/contact-details-form-fields/managed-contact-details-form-fields.jsx

```diff
@@ -243,7 +241,7 @@ export class ManagedContactDetailsFormFields extends Component {
 		return (
 			<div className="contact-details-form-fields__row">
 				<Input
-					label={ this.props.translate( 'Alternate Email Address' ) }
+					label={ this.props.translate( 'Alternate email address' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.